ui/src/main/webapp/assets/css/global-custom.css

```diff
@@ -148,3 +148,15 @@
 #tbl-leaderboard tr {
     cursor: pointer;
 }
+
+#tbl-leaderboard tbody tr :nth-child(3){
+    font-family: 'Source Code Pro', monospace;
+}
+
+#tbl-leaderboard tbody tr :nth-child(4){
+    font-family: 'Source Code Pro', monospace;
+}
+
+#tbl-score-history tbody tr :nth-child(1){
+    font-family: 'Source Code Pro', monospace;
+}
```

Comment:
You can add all these 3 into a single line using commas

#tbl-leaderboard tbody tr :nth-child(3), #tbl-leaderboard tbody tr :nth-child(4) {

Reply:
Shall we introduce a class instead?
The column order might be changed in the future.

Reply:
reffer readme file to setup the project